meinberlin/apps/dashboard2/forms.py

```diff
@@ -49,7 +49,61 @@ class Meta:
         }
 
 
-class ProjectBasicForm(forms.ModelForm):
+def _make_fields_required(fields, required):
+    if required:
+        for name, field in fields:
+            if required == '__all__' or name in required:
+                field.required = True
```

Comment:
>  Is it officially supported?
i'd say yes. the properties are public (non underscored) and from a OOP perspective it is definitively valid to change an objects state by changing its public properties. 

> Are we sure that the same field instance is not reused somewhere else?
yes, the method is invoked with the fields from a concrete form object. this object has it's own list of form fields which it can edit in any way it wants. see the comment at the django code:
https://github.com/django/django/blob/master/django/forms/forms.py#L91